EmployeesSalary/Program.cs

```diff
@@ -50,3 +50,9 @@
 {
     Console.WriteLine("Employee " + employee.Name + "/" + employee.Type + ". Joined in: " + employee.JoinDate + " BaseSalary: " + employee.BaseSalary + ". Total Salary: " + employee.CalculateSalary());
 }
+
+foreach (Employee employee in new[] {manager, salesman, secretary})
+{
+    Console.WriteLine("Emplyee " + employee.Name + " Base Salary: " + employee.BaseSalary + 
+        " Join in : " + employee.JoinDate + " Bouns = " + employee.CalculateBouns());
```

Comment:
How the bonus will be added to the monthly salary? 
the bonus value should be added to the return of Calculate Salary function.